src/sage/rings/polynomial/polynomial_quotient_ring.py

```diff
@@ -1262,6 +1262,69 @@ def polynomial_ring(self):
         """
         return self.__ring
 
+    def free_module(self, base=None, basis=None, map=True):
+        """
+        See :meth:`sage.categories.rings.Rings.ParentMethods.free_module`.
+
```

Comment:
```suggestion
        r"""
        Return a free module `V` over the specified subring together with
        maps to and from `V`.

        See :meth:`sage.categories.rings.Rings.ParentMethods.free_module`.

```
While it duplicates some doc, it is far more useful to have at least something when using `?` than a redirect.